src/settings-ui/Settings.UI/SettingsXAML/Controls/MetadataDotSeparator.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<UserControl
+    x:Class="Microsoft.PowerToys.Settings.UI.Controls.MetadataDotSeparator"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    d:DesignHeight="300"
+    d:DesignWidth="400"
+    mc:Ignorable="d">
+
+    <TextBlock
+        AutomationProperties.Name=","
+        IsTabStop="False"
+        Style="{ThemeResource SecondaryTextStyle}"
+        Text="&#x2022;" />
+</UserControl>
```

Comment:
Is this user control really needed? Seems a bit overkill since it contains only a text block.

Reply:
After reworking some code this morning, no. Not needed anymore.

Reply:
I think it will be a "good" kind of code duplication, but, I am up to another review for this.

Reply:
@nieks9001
Thoughts? Can we have something like a code snippet template that is not complete a user control?